cluster/gce/util.sh

```diff
@@ -477,7 +477,7 @@ function make-gcloud-network-argument() {
     ret="${ret},aliases=pods-default:${alias_size}"
     ret="${ret} --no-can-ip-forward"
   else
-    ret="--network ${network}"
+    ret="--subnet=${network}"
```

Comment:
Right. Somehow I forgot about the case when people may want to start cluster in pre-existing legacy network. Will fix.